Directory.Build.props

```diff
@@ -1,7 +1,8 @@
 <Project>
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
Consider only build for `net8.0` at this timeframe in CI by checking `ContinuousIntegrationBuild` since most contributors likely don't have .NET 8 SDK.